src/libsync/abstractnetworkjob.cpp

```diff
@@ -463,13 +463,89 @@ QString networkReplyErrorString(const QNetworkReply &reply)
+    QString userFriendlyMessage;
+    switch (httpStatus) {
+        case 400: //Bad Request
+            userFriendlyMessage = QObject::tr("We couldn’t process your request. Please try syncing again later. If this keeps happening, contact your server administrator for help.");
+            break;
+        case 401: //Unauthorized
+            userFriendlyMessage = QObject::tr("You need to sign in to continue. If you have trouble with your server credentials, please reach out to your server administrator.");
+            break;
+        case 403: //Forbidden
+            userFriendlyMessage = QObject::tr("You don’t have access to this resource. If you think this is a mistake, please contact your server administrator.");
+            break;
+        case 404: //Not Found
+            userFriendlyMessage = QObject::tr("We couldn’t find what you were looking for. It might have been moved or deleted. If you need help, contact your server administrator.");
+            break;
+        case 407: //Proxy Authentication Required
+            userFriendlyMessage = QObject::tr("It seems you are using a proxy that required authentication. Please check your proxy settings and credentials. If you need help, contact your server administrator.");
+            break;
+        case 408: //Request Timeout
+            userFriendlyMessage = QObject::tr("The request is taking longer than usual. Please try syncing again. If it still doesn’t work, reach out to your server administrator.");
+            break;
+        case 409: //Conflict
+            userFriendlyMessage = QObject::tr("Server files changed while you were working. Please try syncing again. Contact your server administrator if the issue persists.");
+            break;
+        case 410: //Gone
+            userFriendlyMessage = QObject::tr("This folder or file isn’t available anymore. If you need assistance, please contact your server administrator.");
+            break;
+        case 412: //Precondition failed
+            userFriendlyMessage = QObject::tr("The request could not be completed because some required conditions were not met. Please try syncing again later. If you need assistance, please contact your server administrator.");
+            break;
+        case 413: //Payload Too Large
+            userFriendlyMessage = QObject::tr("The file is too big to upload. You might need to choose a smaller file or contact your server administrator for assistance.");
+            break;
+        case 414: //URI Too Long
+            userFriendlyMessage = QObject::tr("The address used to make the request is too long for the server to handle. Please try shortening the information you’re sending or contact your server administrator for assistance.");
+            break;
+        case 415: //Unsupported Media Type
+            userFriendlyMessage = QObject::tr("This file type isn’t supported. You might need to try a different file or contact your server administrator for assistance.");
```

Comment:
> You might need to try a different file

isn't very satisfying :) What if I want this file!?

I think it'd be better to drop that part here.